mage_ai/cache/block.py

```diff
@@ -160,8 +169,10 @@ def __filter(
             ) -> bool:
                 pipeline_dict = cache_dict.get('pipeline') or {}
 
-                return pipeline_dict.get('uuid') != pipeline_uuid and \
-                    pipeline_dict.get('repo_path') == repo_path
+                return pipeline_dict.get('uuid') != pipeline_uuid or (
```

Comment:
Fix caching issue that was improperly excluding pipelines from other projects that were using a block file.